knpu/facebook2.rst

```diff
@@ -405,28 +399,28 @@ redirecting them to re-authorize.
 I'm dumping the result of the API request, so let's logout and try the process. 
 But first, reset the database so that it doesn't find our existing user:
 
-.. code-block:: bash
-
-    rm data/topcluck.sqlite
+```bash
+rm data/topcluck.sqlite
+```
 
 When we login with Facebook, we hit the dump, which holds a lot of nice information
 about the user:
 
-.. code-block:: text
-
-    array (size=12)
-      'id' => string '100002910877036' (length=15)
-      'name' => string '...' (length=17)
-      'first_name' => string '...' (length=10)
-      'last_name' => string '...' (length=6)
-      ...
+```text
+array (size=12)
+  'id' => string '100002910877036' (length=15)
+  'name' => string '...' (length=17)
+  'first_name' => string '...' (length=10)
+  'last_name' => string '...' (length=6)
+  ...
+```
 
 We're allowed to ask for this information because when we redirect the user
-for authorization, we're asking for the ``email`` scope. Let's update the
-``findOrCreateUser`` method to use this data.
+for authorization, we're asking for the `email` scope. Let's update the
+`findOrCreateUser` method to use this data.
 
-First, change ``findUserByCOOPId`` to ``findUserByFacebookId``, which is
-a shortcut method in my app to find a user by the  ``facebookUserId`` column::
+First, change `findUserByCOOPId` to `findUserByFacebookId`, which is
+a shortcut method in my app to find a user by the  `facebookUserId` column::
```

Comment:
`findOrCreateUser()`, `findUserByCOOPId()` and `findUserByFacebookId()` with `:` at the end and php code block on the next line